r3bbase/R3BTprevTnext.h

```diff
@@ -0,0 +1,50 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2023 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+#pragma once
+
+#include "FairTask.h"
+#include "R3BMSOffsetPar.h"
+
+class TClonesArray;
+class R3BMSOffsetPar;
+class R3BEventHeader;
+class R3BSamplerMappedData;
+class R3BTprevTnext : public FairTask
+{
+  public:
+    /** Default constructor **/
+    R3BTprevTnext();
+
+    /** Standard constructor **/
+    explicit R3BTprevTnext(const TString& name, Int_t iVerbose = 1);
+
+    /** Virtual method Init **/
+    InitStatus Init() override;
+
+    /** Virtual method Exec **/
+    void Exec(Option_t* opt) override;
+
+    void SetOne_hit(Double_t One_hit_position) { fOne_hit = One_hit_position; }
+    void SetDelta_clk(Double_t clock) { fDelta_clk = clock; }
+
+  private:
+    R3BMSOffsetPar* fMSOffsetPar = nullptr;
+    TClonesArray* fSamplerMapped = nullptr;
+    TClonesArray* fSamplerMSMapped = nullptr;
+    R3BEventHeader* fR3BEventHeader = nullptr; /**< Event header - input data. */
+    Double_t fDelta_clk = 1.0;
+    Double_t fOne_hit = 3e6;
```

Comment:
``=2e3; // 2us``

If the user does not provide a safe value, assume a smallish value which will make them reject more events rather than a gargantuan value which will see them include these hits in every cut.